financial-connections/src/main/java/com/stripe/android/financialconnections/features/manualentry/ManualEntryViewModel.kt

```diff
@@ -31,14 +36,17 @@ internal class ManualEntryViewModel @Inject constructor(
 ) : MavericksViewModel<ManualEntryState>(initialState) {
 
     init {
-        logErrors()
+        observeAsyncs()
         observeInputs()
         suspend {
```

Comment:
is there any preference on using `suspend` vs. using `viewModelScope`?

Reply:
in `Mavericks`, we use `suspend` to be able to use the `execute`, that internally still uses `viewModelScope`.

https://airbnb.io/mavericks/#/async?id=using-async-for-asynchronous-actions-network-db-etc-with-execute

When you call execute on the suspend block, it will begin executing it, immediately emit Loading, and then emit Success or Fail when it succeeds, emits a new value, or fails.